index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins" />
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Crete Round" />
+        
+    </head>
+    <body>
+        <header class="section-1">
+            <nav class="nav-bar">
+                    
+                    <a class="logo-name">DANIEL OCHUBA</a>
+                    
+                    <div class="menu-button">
+                        <div class="toggle"></div>
+                        <div class="toggle"></div>
+                        <div class="toggle"></div>
+                    </div>
+                    
+                </nav>
+            </header>
+
+        <section>
+            <div class="section-2">
+                <h1 class="primary-text">Hey There. I’m Daniel</h1>
+                <h5 class="supporting-text">I’am a Software Developer</h5>
+                <p class="description">I can help you build a product ,
```

Comment:
- [ ] [OPIONAL] This h5 should be part of the p tag too or you can keep it separate but it should be a p tag after all, make sure you update this comment as it is marked options because the previous reviewer did not mention it